bindings/python/RobotDynamicsEstimator/src/RobotDynamicsEstimator.cpp

```diff
@@ -0,0 +1,223 @@
+/**
+ * @file RobotDynamicsEstimator.cpp
+ * @authors Ines Sorrentino
+ * @copyright 2023 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the BSD-3-Clause license.
+ */
+
+#include <pybind11/pybind11.h>
+#include <pybind11/eigen.h>
+#include <pybind11/stl.h>
+
+#include <iDynTree/Model/Model.h>
+
+#include <BipedalLocomotion/System/Advanceable.h>
+#include <BipedalLocomotion/ParametersHandler/IParametersHandler.h>
+#include <BipedalLocomotion/System/VariablesHandler.h>
+#include <BipedalLocomotion/RobotDynamicsEstimator/RobotDynamicsEstimator.h>
+#include <BipedalLocomotion/RobotDynamicsEstimator/KinDynWrapper.h>
+#include <BipedalLocomotion/RobotDynamicsEstimator/SubModel.h>
+#include <BipedalLocomotion/Conversions/ManifConversions.h>
+
+#include <BipedalLocomotion/bindings/RobotDynamicsEstimator/RobotDynamicsEstimator.h>
+#include <BipedalLocomotion/bindings/System/Advanceable.h>
+#include <BipedalLocomotion/bindings/System/VariablesHandler.h>
+
+namespace BipedalLocomotion
+{
+namespace bindings
+{
+namespace RobotDynamicsEstimator
+{
+
+void CreateRobotDynamicsEstimator(pybind11::module& module)
+{
+    namespace py = ::pybind11;
+
+    using namespace BipedalLocomotion::System;
+    using namespace BipedalLocomotion::ParametersHandler;
+    using namespace BipedalLocomotion::Estimators::RobotDynamicsEstimator;
+
+    py::class_<RobotDynamicsEstimatorInput>(module, "RobotDynamicsEstimatorInput")
+        .def(py::init())
+        .def_readwrite("basePose", &RobotDynamicsEstimatorInput::basePose)
+        .def_readwrite("baseVelocity", &RobotDynamicsEstimatorInput::baseVelocity)
+        .def_readwrite("baseAcceleration", &RobotDynamicsEstimatorInput::baseAcceleration)
+        .def_readwrite("jointPositions", &RobotDynamicsEstimatorInput::jointPositions)
+        .def_readwrite("jointVelocities", &RobotDynamicsEstimatorInput::jointVelocities)
+        .def_readwrite("motorCurrents", &RobotDynamicsEstimatorInput::motorCurrents)
+        .def_readwrite("ftWrenches", &RobotDynamicsEstimatorInput::ftWrenches)
+        .def_readwrite("linearAccelerations", &RobotDynamicsEstimatorInput::linearAccelerations)
+        .def_readwrite("angularVelocities", &RobotDynamicsEstimatorInput::angularVelocities)
+        .def_readwrite("friction", &RobotDynamicsEstimatorInput::friction);
+
+    py::class_<RobotDynamicsEstimatorOutput>(module, "RobotDynamicsEstimatorOutput")
+            .def(py::init())
+            .def_readwrite("ds", &RobotDynamicsEstimatorOutput::ds)
+            .def_readwrite("tau_m", &RobotDynamicsEstimatorOutput::tau_m)
+            .def_readwrite("tau_F", &RobotDynamicsEstimatorOutput::tau_F)
+            .def_readwrite("ftWrenches", &RobotDynamicsEstimatorOutput::ftWrenches)
```

Comment:
```suggestion
            .def_readwrite("ft_wrenches", &RobotDynamicsEstimatorOutput::ftWrenches)
```

the same applies for all the others https://ami-iit.github.io/bipedal-locomotion-framework/python-additional-info.html#autotoc_md0

Reply:
Fixed!